components/crossplane-config/staging/provider-config.yaml

```
@@ -0,0 +1,13 @@
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: eaas-stage-kubernetes-provider-config
+  api-server: https://api.kflux-stg-es01.21tc.p1.openshiftapps.com:6443
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-config
+      name: eaas-stage-kubeconfig
```

Please include the `ExternalSecret` that generates this secret.

---

Nit: since the secret will include more than just a kubeconfig and the env info is redundant (we always know the cluster/env it's deployed to).
```suggestion
      name: eaas-cluster
```